target/linux/ath79/image/generic.mk

```diff
@@ -250,6 +263,11 @@ define Device/compex_wpj531
   IMAGE_SIZE := 16128k
   DEVICE_TITLE := Compex WPJ531 (16MB flash)
   BOARDNAME := WPJ531
+  IMAGES += cpximg-7A03.img cpximg-7A04.img cpximg-7A06.img cpximg-7A07.img
```

Comment:
Is this supposed to be ".bin" or am I misunderstanding something here?

Reply:
oops